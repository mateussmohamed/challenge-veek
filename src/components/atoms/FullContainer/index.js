import styled from 'styled-components'

const FullContainer = styled.section`
  width: 100%;
  display: flex;

  @media only screen and (min-width: 0em) {
    height: 100vh;
  }
  @media only screen and (min-width: 48em) {
    height: 100vh;
  }
`

export default FullContainer
