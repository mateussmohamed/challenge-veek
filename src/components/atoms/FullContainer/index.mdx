---
name: FullContainer
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import FullContainer from './index'

# FullContainer
<PropsTable of={FullContainer} />

<Playground>
  <FullContainer>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae tempus odio. Nam gravida diam eros, non ultricies velit varius ut.
  </FullContainer>
</Playground>
