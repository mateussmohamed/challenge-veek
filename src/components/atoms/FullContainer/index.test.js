import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import FullContainer from './index'

describe('[<FullContainer />]', () => {
  const wrap = (props = {}) => shallow(<FullContainer {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
