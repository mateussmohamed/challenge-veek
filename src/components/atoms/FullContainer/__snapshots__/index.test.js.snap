// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<FullContainer />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "FullContainer-sc-16e7xah-0",
        "isStatic": true,
        "rules": Array [
          "width:100%;display:flex;@media only screen and (min-width:0em){height:100vh;}@media only screen and (min-width:48em){height:100vh;}",
        ],
      },
      "displayName": "FullContainer",
      "render": [Function],
      "styledComponentId": "FullContainer-sc-16e7xah-0",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
